packages/psss/src/api/mutations.js

```diff
@@ -5,20 +5,27 @@
 
 import { queryCache, useMutation } from 'react-query';
 import { saveSiteReport } from './requests';
+import { post } from './api';
 import { FakeAPI } from './FakeApi';
 
 export const useSaveSiteReport = params =>
   useMutation(saveSiteReport, {
     onSuccess: () => queryCache.invalidateQueries('country-weeks', params),
   });
 
-export const useConfirmWeeklyReport = orgUnit =>
-  useMutation(() => console.log('post to confirmWeeklyReport...'), {
-    onSuccess: () => {
-      queryCache.invalidateQueries(`weeklyReport/${orgUnit}`);
-      queryCache.invalidateQueries(`confirmedWeeklyReport/${orgUnit}`);
+export const useConfirmWeeklyReport = (orgUnit, period) =>
+  useMutation(
+    () =>
+      post(`confirmedWeeklyReport/${orgUnit}`, {
+        params: { week: period },
+      }),
+    {
+      onSuccess: () => {
+        queryCache.invalidateQueries(`weeklyReport/${orgUnit}`);
+        queryCache.invalidateQueries(`confirmedWeeklyReport/${orgUnit}`);
```

Comment:
Should this also invalidate `confirmedWeeklyReport`?

Reply:
Yes unfortunately it needs to because I use that to calculate the week status. There is a task in the backlog to create an indicator for the status and have it included in the server response which I think is more appropriate in the long term. However I thought it would be better to have something working than not for MVP.

Reply:
ps. by week status, I mean `overdue`, `resubmit`, `submitted`

Reply:
Oh sorry, I meant should this invalidate the query for all countries in addition to invalidating the query for a single country? 
```suggestion
        queryCache.invalidateQueries(`confirmedWeeklyReport/${orgUnit}`);
        queryCache.invalidateQueries(`confirmedWeeklyReport`);        
```

Reply:
ah I see what you mean now. Yes you are correct - for when the user goes back to the home page. Nice